specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/machineLearningServices.json

```diff
@@ -1986,627 +2419,1417 @@
+                },
+                "paths": {
+                  "type": "array",
+                  "description": "List of files expanded from a file GLOB specified",
+                  "items": {
+                    "type": "string"
+                  },
+                  "readOnly": true
+                },
+                "sqlDataPath": {
+                  "type": "object",
+                  "description": "Sql Query/Table/Stored Procedure details.",
+                  "properties": {
+                    "sqlTableName": {
+                      "readOnly": true,
+                      "type": "string",
+                      "description": "SQL table name"
+                    },
+                    "sqlQuery": {
+                      "readOnly": true,
+                      "type": "string",
+                      "description": "SQL query"
+                    },
+                    "sqlStoredProcedureName": {
+                      "readOnly": true,
+                      "type": "string",
+                      "description": "SQL storedProcedure name"
+                    },
+                    "queryTimeout": {
+                      "readOnly": true,
+                      "type": "integer",
+                      "format": "int64",
+                      "description": "SQL query timeout. Unit in seconds."
+                    }
+                  }
+                },
+                "httpUrl": {
+                  "readOnly": true,
+                  "type": "string",
+                  "description": "HTTP URL."
+                },
+                "additionalProperties": {
+                  "description": "Additional Properties.",
+                  "type": "object",
+                  "additionalProperties": {
+                    "type": "object"
+                  }
+                },
+                "partitionFormat": {
+                  "readOnly": true,
+                  "type": "string",
+                  "description": "Specify the partition format of path. Defaults to None."
+                },
+                "partitionFormatIgnoreError": {
+                  "type": "boolean",
+                  "description": "Whether or not to ignore unmatched path.",
+                  "readOnly": true
+                }
+              }
+            },
+            "partitionFormatInPath": {
+              "type": "boolean",
+              "description": "Indicates how the source data is partitioned. This is defined to filter on a range of partitioned data before performing actions or materialization.",
+              "readOnly": true
+            },
+            "notes": {
+              "type": "string",
+              "description": "Summary of Definition changes.",
+              "readOnly": true
+            },
+            "etag": {
+              "type": "string",
+              "description": "eTag description",
+              "readOnly": true
+            },
+            "createdTime": {
```

Comment:
new system data ARM top level property mUST be leveraged for such information. Details here - https://armwiki.azurewebsites.net/api_contracts/ResourceSystemData.html?q=systemdata

Reply:
Actually these are properties from documentDB for these resources similar to model object here-

https://github.com/Azure/azure-rest-api-specs/blob/3aa6a51349b07a7b8672ee0fdf8ba5dbeee31153/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/machineLearningServices.json#L4281

Could Dataset/Datastore resources follow similar pattern as computes & services resource since all of them all provided by machine learning workspace resource?